DEPLOYMENT.md

```diff
@@ -0,0 +1,58 @@
+# Steps
+
+When all tests are done and successful and PR is approved by codeowners, follow these steps:
+
+1. Select "Squash and merge" to merge branch into default branch (master/main).
+
+
+2. A prompt for writing merge commit message will pop up.
+
+
+3. Find the title of the pull request already pre-filled in the merge commit title, or copy and paste 
+the title if not.
+
+
+4. Append version increment value `( major | minor | patch )` to specify what kind of release is to be created.
+
+
+5. Fill in markdown formatted changelog in merge commit comment details:
+
+` ### Added `
+
+` ### Changed `
+
+` ### Fixed `
+
+6. Review the details once again and merge the branch into master.
+
+
+7. Wait for GitHub actions to process the event, bump version, create release, publish to Dockerhub and PyPi.
+
+
+8. Deploy master to stage:
+    1. Log in to appropriate server `ssh <server.scilifelab.se>`
```

Comment:
can we make this explicit as to what server is the appropriate one?

Reply:
Fixed